src/Security/Authentication/OAuth/src/Events/OAuthCreatingTicketContext.cs

```diff
@@ -33,7 +33,7 @@ public class OAuthCreatingTicketContext : ResultContext<OAuthOptions>
             OAuthOptions options,
             HttpClient backchannel,
             OAuthTokenResponse tokens)
-            : this(principal, properties, context, scheme, options, backchannel, tokens, user: new JObject())
+            : this(principal, properties, context, scheme, options, backchannel, tokens, user: JsonDocument.Parse(string.Empty))
```

Comment:
FYI `JsonDocument.Parse(string.Empty)` and `new JObject()` aren't necessarily equivalent.

`new JObject()` = an empty JSON object

`JsonDocument.Parse("{}")` would be closer. I don't know if it matters in this case.

Reply:
`JsonDocument.Parse(string.Empty)`

I don't think this will work.

An empty json string is considered invalid and hence we throw an exception:
https://github.com/dotnet/corefx/blob/5197c8dc78adcb130a08944724f1b49c3765a4b1/src/System.Text.Json/tests/JsonDocumentTests.cs#L1204

Also, given that JsonDocument is readonly, I don't know if it is useful to create an empty JsonDocument.

`JsonDocument.Parse("{}")` <= this will work, but it is odd to see.

cc @bartonjs

Reply:
Changed to `"{}"`.  Having an empty document placeholder can make usage simpler by being able to exclude a lot of null checks.